--- /etc/rsyslog.conf	2019-10-18 15:48:29.000000000 +0000
+++ /root/rsyslog.docker.centos7.conf	2020-03-12 10:47:58.365192000 +0000
@@ -4,10 +4,13 @@
 # If you experience problems, see http://www.rsyslog.com/doc/troubleshoot.html

 #### MODULES ####
+module(load="imuxsock")
+module(load="omstdout")
+*.* :omstdout:

 # The imjournal module bellow is now used as a message source instead of imuxsock.
-$ModLoad imuxsock # provides support for local system logging (e.g. via logger command)
-$ModLoad imjournal # provides access to the systemd journal
+#$ModLoad imuxsock # provides support for local system logging (e.g. via logger command)
+#$ModLoad imjournal # provides access to the systemd journal
 #$ModLoad imklog # reads kernel messages (the same are read from journald)
 #$ModLoad immark  # provides --MARK-- message capability

@@ -37,10 +40,10 @@

 # Turn off message reception via local log socket;
 # local messages are retrieved through imjournal now.
-$OmitLocalLogging on
+#$OmitLocalLogging on

 # File to store the position in the journal
-$IMJournalStateFile imjournal.state
+#$IMJournalStateFile imjournal.state


#### RULES ####
